src/core/config/transformer/rename_args.rs

```diff
@@ -0,0 +1,126 @@
+use indexmap::IndexMap;
+
+use crate::core::config::{Config, Resolver};
+use crate::core::valid::Valid;
+use crate::core::Transform;
+
+/// A transformer that renames existing args by replacing them with suggested
+/// names.
+pub struct RenameArgs(IndexMap<String, Vec<(String, String)>>);
```

Comment:
We are renaming fields and not arguments:
```suggestion
pub struct RenameField(IndexMap<(TypeName, FieldName), FieldName>);
```

Similarly if we were renaming Args it should look like 

```rust
pub struct RenameArgs(IndexMap<(TypeName, FieldName, ArgName), ArgName>);
```